examples/shader/compute_shader_game_of_life.rs

```diff
@@ -222,10 +222,12 @@ impl render_graph::Node for GameOfLifeNode {
         // if the corresponding pipeline has loaded, transition to the next stage
         match self.state {
             GameOfLifeState::Loading => {
-                if let CachedPipelineState::Ok(_) =
-                    pipeline_cache.get_compute_pipeline_state(pipeline.init_pipeline)
-                {
-                    self.state = GameOfLifeState::Init;
+                match pipeline_cache.get_compute_pipeline_state(pipeline.init_pipeline) {
+                    CachedPipelineState::Ok(_) => {
+                        self.state = GameOfLifeState::Init;
+                    }
+                    CachedPipelineState::Err(err) => panic!("{err}"),
```

Comment:
Can you add a comment here warning users that they should copy-paste the shader too?